Admin/AbstractAdmin.php

```diff
@@ -2843,6 +2838,25 @@ public function getDashboardActions()
     }
 
     /**
+     * Setting to true will enable mosaic button for
+     * the admin screen.
```

Comment:
Well in that case, you didn't forget, you are more than respecting it ;) We tried to find a solution, but there is no fixer for that. See #3724